src/Symfony/Component/OptionsResolver/Tests/OptionsResolver2Dot6Test.php

```diff
@@ -1557,4 +1522,57 @@ public function testCountFailsOutsideResolve()
 
         count($this->resolver);
     }
+
+    /**
+     * @dataProvider getFormatValueData
+     * @expectedException \Symfony\Component\OptionsResolver\Exception\InvalidOptionsException
+     */
+    public function testFormatValue($actualType, $allowedType, $exceptionMessage)
+    {
+        $this->resolver->setDefined('option');
+        $this->resolver->setAllowedTypes('option', $allowedType);
+
+        $this->resolver->resolve(array('option' => $actualType));
+    }
+
+    public function getFormatValueData()
```

Comment:
please put this below `testResolveFailsIfInvalidType` and rename it to `provideInvalidTypes`